pureport/resource_pureport_google_cloud_connection.go

```diff
@@ -61,8 +60,8 @@ func resourceGoogleCloudConnection() *schema.Resource {
 		Schema: connection_schema,
 
 		Timeouts: &schema.ResourceTimeout{
-			Create: schema.DefaultTimeout(6 * time.Minute),
-			Delete: schema.DefaultTimeout(6 * time.Minute),
+			Create: schema.DefaultTimeout(connection.CreateTimeout),
+			Delete: schema.DefaultTimeout(connection.CreateTimeout),
```

Comment:
This should be DeleteTimeout

Reply:
Good catch! It's like that everywhere. I need to finish my coffee before I review anything. :-(

Reply:
Should be fixed.